common/pb/admin_commands.proto

```diff
@@ -26,3 +27,13 @@ message Command_ReloadConfig {
 		optional Command_ReloadConfig ext = 1002;
 	}
 }
+
+message Command_AdjustMod {
```

Comment:
Hold on a second.  This brings up a good point.  The response container is going to be processed client side.  Are you suggesting I then make a second server call to do a simply client notifiction?